### ISSUE TITLE
[AUR] infisical-bin: missining .LICENSE in the package

### ISSUE THREAD

**Issue by gipo355** [Infisical/infisical#563]
### Describe the bug
The AUR package infisical-bin is broken, looks like it's missing the LICENSE file to complete the install for package 
https://github.com/Infisical/infisical/releases/download/infisical-cli/v0.4.4/infisical_0.4.4_linux_amd64.tar.gz

### To Reproduce
on any arch distro for amd64: yay infisical-bin

### Screenshots
```
=> Starting package()...
install: cannot stat './LICENSE': No such file or directory
==> ERROR: A failure occurred in package().
    Aborting...
 -> error making: infisical-bin-exit status 4
 -> Failed to install the following packages. Manual intervention is required:
infisical-bin - exit status 4
```

### Platform you are having the issue on:
endeavour-os, amd64

**Comment by maidul98** [comment]
Hey @gipo355, i'm taking a look at this

**Comment by gipo355** [comment]
i was able to manually download the source archive it was pointing to and place in bin but the tar.gz is missing the LICENSE file, the package() fn in the PKGBUILD points to it and doesn't find it so it breaks

**Comment by maidul98** [comment]
Yes, this is related to our recent changes with Go Releaser. Happy you found a fix for the time being. I'll provide an update once this is fixed

**Comment by gipo355** [comment]
thank you :pray: amazing project!

**Comment by maidul98** [comment]
discussion started https://github.com/orgs/goreleaser/discussions/3991

**Comment by maidul98** [comment]
This issue has been resolved! Let us know if things are working on your end

**Comment by gipo355** [comment]
Thank you @maidul98! the package now installs! 

Autocompletion for zsh doesn't work out of the box tho unfortunately.

At the moment, `install -Dm644 "./completions/infisical.zsh" "${pkgdir}/usr/share/zsh/site-functions/infisical"` doesn't work for me.

I am able to make it work with those 2 methods:

**solution 1:**
mv `/usr/share/zsh/site-functions/infisical` to `/usr/share/zsh/functions/Completion/Unix/_infisical`

**solution 2:**
mv `/usr/share/zsh/site-functions/infisical` to `/usr/share/zsh/site-functions/_infisical`

either of the above work, it appears that the file must be preceded by an underscore dangle to be recognized  ( every other file has it )

**possible fix**
modify `PKGBUILD` file:
`install -Dm644 "./completions/infisical.zsh" "${pkgdir}/usr/share/zsh/site-functions/_infisical"`


*using `zsh`, `oh-my-zsh` on endeavour-os(arch), amd64*

**Comment by maidul98** [comment]
I have added `_`, let me know if the new version is working for you
